timing/timeMatrixOps.cpp

```diff
@@ -33,7 +33,8 @@ using namespace std;
 using boost::format;
 using namespace boost::lambda;
 
-static boost::variate_generator<boost::mt19937, boost::uniform_real<> > rng(boost::mt19937(), boost::uniform_real<>(-1.0, 0.0));
+static std::mt19937 rng;
+static std::uniform_real_distribution<> rg(-1.0, 0.0);
```

Comment:
maybe call this uniform?

Reply:
Yeah was debating on change the name to be more appropriate.

Reply:
changed `rg` -> `uniform`
`rni` -> `ri` -> `uniform_i`
`rnj` -> `rj` -> `uniform_j`